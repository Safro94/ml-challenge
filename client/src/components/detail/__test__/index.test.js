import React from 'react';

import { render } from '@testing-library/react';

import Detail from '../';

describe('Detail', () => {
	it('should render OK', () => {
		//Arrange
		const item = {
			picture: 'a.jpg',
			title: 'title',
			condition: 'new',
			sold_quantity: 20,
			price: {
				currency: '$',
				amount: 30,
				decimals: 50,
			},
		};

		//Act
		const { container } = render(<Detail item={item} />);

		//Assert
		expect(container).toMatchSnapshot();
	});
});
