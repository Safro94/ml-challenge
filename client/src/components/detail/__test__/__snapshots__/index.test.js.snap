// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Detail should render OK 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="imageContainer"
    >
      <img
        alt="title"
        class="image"
        src="a.jpg"
      />
    </div>
    <div
      class="information"
    >
      <h4
        class="quantity"
      >
        new
         - 
        20
         vendidos
      </h4>
      <h2
        class="title"
      >
        title
      </h2>
      <h1
        class="price price"
      >
        <span
          class="currencyCode"
        >
          $
        </span>
        <span>
          30.50
        </span>
        <span
          class="decimals"
        />
      </h1>
      <div>
        <button
          class="button button"
          type="submit"
        >
          Comprar
        </button>
      </div>
    </div>
  </div>
</div>
`;
